### PR TITLE
Give compiler hint about inline functions

### PR DESCRIPTION
**Pull request by normanmaurer**
Motivation:

Some of the methods are frequently called and so should be inlined if possible.

Modifications:

Give the compiler a hint that we want to inline these methods.

Result:

Better performance if inlined.

### CODE REVIEW

**Comment by normanmaurer** [comment]
@trustin please review

**Comment by buchgr** [comment]
@normanmaurer did you check if this change actually does anything? In my experience compilers completely ignore inline and decide for themselves if something should be inlined or not.

**Comment by normanmaurer** [comment]
@buchgr well it's a good practice, so can't harm (note also the static stuff) :)

File: transport-native-epoll/src/main/c/io_netty_channel_epoll_Native.c [hunk 1]
```diff
@@ -78,25 +78,25 @@ static int socketType;
 static const char* ip4prefix = "::ffff:";
 
 // util methods
-void throwRuntimeException(JNIEnv* env, char* message) {
+static void throwRuntimeException(JNIEnv* env, char* message) {
```
Comment (trustin) [inline transport-native-epoll/src/main/c/io_netty_channel_epoll_Native.c]: These `throw*` methods seem to be a good candidate for inlining. Could you mark them inlined?

**Comment by trustin** [comment]
Looks OK to me. Please address the comment and cherry-pick.  I however wonder it will bring us real benefit. Numbers?

**Comment by normanmaurer** [comment]
cherry-picked into 4.0 (6231113c38d7a2ff55fb7503e8a61fb625840554), 4.1 (7a3d91f43d12eb0b23a65662d26026c6cd451d76) and master (8f61e4cdd67906ce3216da015aec65cd14b18d78)

**Pull request closed without merging**